src/devices/brun_holding.ts

```diff
@@ -0,0 +1,25 @@
+import * as m from "../lib/modernExtend";
+import type {DefinitionWithExtend} from "../lib/types";
+
+export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["Power Control Unit"],
+        model: "Fire Fence",
+        vendor: "Brun Holding",
+        description: "Stove guard for safe cooking",
+        extend: [
+            m.deviceEndpoints({endpoints: {mainSwitch: 1, shortOverride: 2, ota: 10}}),
```

Comment:
```suggestion
            m.deviceEndpoints({endpoints: {main_switch: 1, short_override: 2}}),
```

Snake casing and I don't think the `ota` endpoint is needed.